src/@types/index.d.ts

```diff
@@ -0,0 +1 @@
+type Nullable<T> = T | null
```

Comment:
What is the value of using `Nullable<T>` over just writing `T | null` directly?

Reply:
I see sometimes, defining object with many keys, such as...
```ts
const myOptionalObject: {
  prop1: type1,
  prop2: type2,
  .
  .
  propN: typeN
} | null = null
```
This will worth for `Nullable<...>`

How do you think about this?

Reply:
Thanks for the explanation!

Now I see. I agree that it looks easier to read, but then there becomes many ways to represent the same type. Let’s try using `Nullable<T>` then.